eff/numer.C

```diff
@@ -0,0 +1,115 @@
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_550_560.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_560_570.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_570_580.root");
+  //t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_580_590.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_590_600.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_600_610.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_60_70.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_610_620.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_620_630.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_630_640.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_640_650.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_650_660.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_660_670.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_670_680.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_680_690.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_690_700.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_700_710.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_70_80.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_710_720.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_720_730.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_730_740.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_740_750.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_750_760.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_760_770.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_770_780.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_780_787.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_80_90.root");
+  t->Add("/eos/cms/store/group/phys_bphys/cavalari/skimbig/BuToKee_v2_noreg_90_100.root");
+
+//  TFile f("root/francesca.root");
+//  TTree* t = (TTree*)f.Get("TaPtree");
+  std::cout << "entries: " << t->GetEntries() << std::endl;
+  
+  TH2F numer_reco("numer_reco","numer_reco",10, 0. ,10.,10, 0. ,10.);
+  int n1 = t->Draw("tag_pt:probe_pt>>numer_reco","bmatchMC==1 && analysisBdtO > 8. && mll_fullfit>1.1 && mll_fullfit<6.25","goff");
```

Comment:
This line should be, 

mll_fullfit*mll_fullfit>1.1 && mll_fullfit*mll_fullfit<6.25

as the mll_fullfit is q, not q^2. see, 
https://github.com/CMSBParking/BParkingNANO/blob/b9bcad0b9b56f9513f47b5066b7d42d32289b3b7/BParkingNano/plugins/BToKLLBuilder.cc#L190